client/my-sites/activity/activity-log/index.jsx

```diff
@@ -583,6 +583,7 @@ class ActivityLog extends Component {
 					filter={ filter }
 					isLoading={ ! displayRulesLoaded || ! logsLoaded }
 					isVisible={ ! ( isEmpty( logs ) && isFilterEmpty ) }
+					showTextSearch={ true }
```

Comment:
I don't think we need this here.  The Filterbar component already has a property for explicitly stating what selectors to show.  The Agency portal utilizes it with just `issueType`.  If we want to be explicit here we can pass `selectorTypes={ dateRange: true, actionType: true, textSearch: true }` then use `selectorTypes.textSearch &&` instead of `showTextSearch &&` to filter it out.

Reply:
I guess in the same vein here, is there a compelling reason to not just call this a TextSelector to be in line with the other Filterbar features?

Reply:
Makes sense. Just renamed it to `TextSelector` to make it consistent with other features, and also enabled this selector by default if no `selectorTypes` is passed. Thanks for the feedback!